src/genie/libs/parser/iosxe/show_bgp.py

```diff
@@ -2253,6 +2253,7 @@ def cli(self, address_family='', vrf='', rd='', output=None):
             # Execute command
             show_output = self.device.execute(cmd)
         else:
+            cmd = ''
```

Comment:
thanks for pointing out this. we are using 'cmd' anyway, so better to do this before 'if' like below. please apply the same to other places.
```
    def cli(self, address_family='', vrf='', rd='', output=None):

        cmd = '' # <<<<<< HERE
        if output is None:
            # Build command
            if vrf:
                if address_family:
                    cmd = self.cli_command[0].format(address_family=address_family,
                                                 vrf=vrf)
            elif rd:
                if address_family:
                    cmd = self.cli_command[1].format(address_family=address_family,
                                                 rd=rd)
            elif address_family:
                cmd = self.cli_command[2].format(address_family=address_family)

            else:
                cmd = self.cli_command[3]
            # Execute command
            show_output = self.device.execute(cmd)
        else:
            show_output = output

Reply:
Thanks for the feedback!  I believe this is addressed in ef3aa8f.

Reply:
@supertylerc thanks for the update!